.github/workflows/scala.yml

```diff
@@ -149,9 +149,12 @@ jobs:
     - name: Run tests
       env:
         TEST_API_KEY: ${{ secrets.playgroundApiKey }}
-        TEST_AAD_TENANT_BLUEFIELD: "b86328db-09aa-4f0e-9a03-0136f604d20a"
-        TEST_CLIENT_ID_BLUEFIELD: ${{ secrets.BLUEFIELD_CLIENT_ID }}
-        TEST_CLIENT_SECRET_BLUEFIELD: ${{ secrets.BLUEFIELD_CLIENT_SECRET }}
+        TEST_PROJECT_NAME: playground
+        TEST_BASE_URL: https://api.cognitedata.com
+        TEST_AAD_TENANT: "b86328db-09aa-4f0e-9a03-0136f604d20a"
+        TEST_CLIENT_ID: ${{ secrets.BLUEFIELD_CLIENT_ID }}
+        TEST_CLIENT_SECRET: ${{ secrets.BLUEFIELD_CLIENT_SECRET }}
+        TEST_SCOPES: https://api.cognitedata.com/.default
```

Comment:
Aha ok, they are hardcoded here but I was thinking about running the tests locally.  
Also can you reflect the test setup changes in README so when someone needs to run the tests, they would know what to put inside .env